charts/loki/values.yaml

```diff
@@ -44,6 +44,8 @@ tracing:
   jaegerAgentHost:
 
 config:
+  ## Name of the existing secret containing loki.yaml key and the configuration as value
+  # existingSecret:
```

Comment:
Let's remove the comment here. That helps IDEs to do auto completion.

Reply:
https://github.com/grafana/helm-charts/pull/542